config/engines.yml

```diff
@@ -249,6 +249,16 @@ rubymotion:
   enable_regexps:
   default_ratings_paths:
     - "**.rb"
+
+shellcheck:
+  image: codeclimate/codeclimate-shellcheck
+  description: A static analysis tool for shell scripts
```

Comment:
I can add the period, are you also saying I need to make this a grammatically correct sentence? "ShellCheck is a..."